packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -300,4 +308,23 @@
             right: 16px;
         }
     }
+    &--filter {
+        margin-left: 1em;
```

Comment:
Can you use `rem` instead, unless there is a reason for `em` to be here? Could you also change upgrade to `rem` in other places>